package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "trufflepig",
-  "version": "1.0.2",
+  "version": "1.0.3",
```

Comment:
IMHO, version bumps and changelog entries should not be in a PR where code is touched. Let's imagine two PRs coming in at the same time and changing the version. That'd be a mess at some point. Let's remove it here and either bump it on master or create a PR where we bump it (less preferable).

Reply:
Makes sense 👍 

The commit has been removed, so this should be good to go now.